config/cloudinaryUpload.js

```diff
@@ -0,0 +1,44 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+import express from "express";
+import * as cloudinary from "cloudinary";
+import fs from "fs";
```

Comment:
there is no need to import this module of file manipulation since your are saving your photos to cloude

Reply:
Yes the files are being uploaded to cloudinary but the process needs a little bit of file manipulation, because:

Step 1: the file before being uploaded is temporary stored in the uploads folder

Step 2: if the cloudinary upload is successful the temporary file needs to be deleted from the uploads folder

File manipulation is needed just for deleting the temporary file.

Reply:
so where is the process of storing file as temp?

Reply:
yes, the file manipulation happens on line 30 and 39